CMakeLists.txt

```diff
@@ -209,12 +209,26 @@ if( MIOPEN_BACKEND STREQUAL "HIP" OR MIOPEN_BACKEND STREQUAL "HIPOC")
     endif()
 endif()
 message( STATUS "${MIOPEN_BACKEND} backend selected." )
-# look for and register extractkernel
-find_program(EXTRACTKERNEL_BIN extractkernel
+	
+# look for and register clang-offload-bundler
+find_program(MIOPEN_OFFLOADBUNDLER_BIN clang-offload-bundler
```

Comment:
I think you may have to guard this, because HCC has a clang-offload-bundler binary inside /opt/rocm/hcc/bin, and it can be found if hcc was added to CMAKE_PREFIX_PATH. HCC's objects may not be compatible with clang-offload-bundler.

Reply:
In a ROCm 3.3 system:
aaron@shien:~$ /opt/rocm/hcc/bin/clang-offload-bundler --version
HCC clang-offload-bundler version 10.0.0 (/data/jenkins-workspace/compute-rocm-rel-3.3/external/hcc-tot/llvm-project/clang 1ce0fe5e88b2124494b9500817b4c2c66bdfa5aa) (based on HCC 3.1.20114-6776c83f-1ce0fe5e88b )

Reply:
It should only look there if it goes into that directory. @pfultz2 do yo have any input on this?